master/src/main/scala/Main.scala

```diff
@@ -1,7 +1,43 @@
 package kr.ac.postech.paranode.master
 
+import kr.ac.postech.paranode.core.WorkerMetadata
+import kr.ac.postech.paranode.master.AuxFunction.getWorkerDetails
+import kr.ac.postech.paranode.rpc.MasterServer
+
+import java.net.URL
+import scala.io.Source
+import scala.util.Try
+
 object Main {
   def main(args: Array[String]): Unit = {
-    println("Hello world!")
+    val requestLimit = Try(args(0).toInt).getOrElse {
+      println("Invalid command")
+      return
+    }
+
+    val server = new MasterServer(scala.concurrent.ExecutionContext.global)
+    server.startServer()
+
+    while (server.getRequestCount < requestLimit) {
+      Thread.sleep(1000)
+    }
```

Comment:
I am thinking of introducing observer pattern, e.g. rxJava. Maybe it could be done on next iteration.